controllers/release/release_controller.go

```diff
@@ -57,7 +59,11 @@ func NewReleaseReconciler(client client.Client, logger *logr.Logger, scheme *run
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := r.Log.WithValues("Release", req.NamespacedName)
+	logger := r.Log.WithValues("Release", req.NamespacedName).WithValues("clusterName", req.ClusterName)
+
+	if req.ClusterName != "" {
+		ctx = logicalcluster.WithCluster(ctx, logicalcluster.New(req.ClusterName))
```

Comment:
@davidmogar we don't ever return ctx in this function and it isn't available in `(ctrl.Result, error)` which is the return type, so I don't know how to test this.